src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -136,9 +136,8 @@ public AWSCredentials getCredentials() {
             DefaultAwsRegionProviderChain sdkRegionLookup = new DefaultAwsRegionProviderChain();
             try {
                 clientRegion = sdkRegionLookup.getRegion();
-            }
-            catch(com.amazonaws.SdkClientException e) {
-                LOGGER.log(Level.WARNING,"Could not find default region using SDK lookup.", e);
+            } catch (com.amazonaws.SdkClientException e) {
```

Comment:
or just

```suggestion
            } catch (RuntimeException e) {
```

?

Reply:
🤷‍♂️ `SdkBaseException` works too: https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/SdkBaseException.html

Reply:
Right. FWIW my general take on Java exceptions is: either your call worked or it failed, and if you are bothering to handle failures at some call site, just handle everything unless you have a specific reason to do otherwise.